src/OrbitGl/GraphTrack.h

```diff
@@ -41,6 +41,12 @@ class GraphTrack : public Track {
   void SetLabelUnitWhenEmpty(const std::string& label_unit);
   void SetValueDecimalDigitsWhenEmpty(uint8_t value_decimal_digits);
 
+  void OnTimer(const orbit_client_protos::TimerInfo& timer_info) override;
+  std::vector<std::shared_ptr<TimerChain>> GetAllChains() const override;
+  std::vector<std::shared_ptr<TimerChain>> GetAllSerializableChains() const override {
+    return GetAllChains();
```

Comment:
I'm not sure why we have both GetAllSerializableChains and GetAllChains

Reply:
To summarize our offline discussion:
- `GetAllChains` returns all the timer chains in the track while `GetAllSerializableChains` returns the timer chains we would like to save in the capture file;
- For most types of tracks, `GetAllSerializableChains` just return the result of `GetAllChains`, but in the `AsyncTrack`, `GetAllSerializableChains` returns an empty vector to avoid serializing the async timer twice (the timers in the track already serialized on their initial thread tracks).